tests/bluetooth/tester/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,23 @@
+# CONFIG_TEST enforces minimal logging, which we don't want
+CONFIG_TEST=n
+
+CONFIG_ASSERT=y
+# Enable the option below to measure stack usage
+#CONFIG_INIT_STACKS=y
+CONFIG_THREAD_NAME=y
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_RTT_CONSOLE=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_MINIMAL=n
```

Comment:
Which warning is this removing?

Reply:
1) The app (btp protocol) does not work properly when not setting "CONFIG_LOG_MINIMAL=n" for nrf5340. For nrf5340, that config is  a must.
2) For nrf52840, it works with or without setting the same config. But when setting, it removes the warning.:
`west build -b nrf52840dk_nrf52840 -p always -- -DOVERLAY_CONFIG="privacy.conf"`
Here privacy.conf can be enforce_mitm.conf which we require for different test cases in PTS testing. Those configs are based on prj.conf as default configuration and add additional configuration depending on the test case. So if you want to repeat the warnings, you can replace privacy.conf with prj.conf as well. When building with additional configuration with overlay_config parameter and not setting "CONFIG_LOG_MINIMAL=n" creates the following warnings for nrf52840:

```
[ncs-int:tester] west build -b nrf52840dk_nrf52840 -p always -- -DOVERLAY_CONFIG="prj.conf"
-- west build: making build dir /work/ncs/zephyr/tests/bluetooth/tester/build pristine
-- west build: generating a build system
Including boilerplate (Zephyr base): /work/ncs/zephyr/cmake/app/boilerplate.cmake
-- Application: /work/ncs/zephyr/tests/bluetooth/tester
-- Zephyr version: 2.4.99 (/work/ncs/zephyr)
-- Found Python3: /usr/local/bin/python3.8 (found suitable exact version "3.8.2") found components: Interpreter 
-- Found west (found suitable version "0.9.0", minimum required is "0.7.1")
-- Board: nrf52840dk_nrf52840
-- Cache files will be written to: /home/.cache/zephyr
-- Found toolchain: zephyr (/opt/zephyr-sdk)
-- Found dtc: /opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.5.0", minimum required is "1.4.6")
-- Found BOARD.dts: /work/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts
-- Found devicetree overlay: /work/ncs/zephyr/tests/bluetooth/tester/nrf52840dk_nrf52840.overlay
-- Generated zephyr.dts: /work/ncs/zephyr/tests/bluetooth/tester/build/zephyr/zephyr.dts
-- Generated devicetree_unfixed.h: /work/ncs/zephyr/tests/bluetooth/tester/build/zephyr/include/generated/devicetree_unfixed.h
-- Generated device_extern.h: /work/ncs/zephyr/tests/bluetooth/tester/build/zephyr/include/generated/device_extern.h
Parsing /work/ncs/zephyr/Kconfig
Loaded configuration '/work/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840_defconfig'
Merged configuration '/work/ncs/zephyr/tests/bluetooth/tester/prj.conf'
Merged configuration '/work/ncs/zephyr/tests/bluetooth/tester/boards/nrf52840dk_nrf52840.conf'
Merged configuration 'prj.conf'
Configuration saved to '/work/ncs/zephyr/tests/bluetooth/tester/build/zephyr/.config'
Kconfig header saved to '/work/ncs/zephyr/tests/bluetooth/tester/build/zephyr/include/generated/autoconf.h'

warning: LOG_PROCESS_THREAD_STACK_SIZE (defined at subsys/logging/Kconfig:207) was assigned the
value '1024' but got the value ''. Check these unsatisfied dependencies: LOG_PROCESS_THREAD (=n),
(!LOG_MINIMAL) (=n). See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_LOG_PROCESS_THREAD_STACK_SIZE.html
and/or look up LOG_PROCESS_THREAD_STACK_SIZE in the menuconfig/guiconfig interface. The Application
Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of
the manual might be helpful too.


warning: LOG_BUFFER_SIZE (defined at subsys/logging/Kconfig:222) was assigned the value '4096' but
got the value ''. Check these unsatisfied dependencies: (!LOG_MINIMAL) (=n). See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_LOG_BUFFER_SIZE.html and/or look up
LOG_BUFFER_SIZE in the menuconfig/guiconfig interface. The Application Development Primer, Setting
Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful
too.


warning: LOG_BACKEND_RTT (defined at subsys/logging/Kconfig:329) was assigned the value 'y' but got
the value 'n'. Check these unsatisfied dependencies: (!LOG_MINIMAL) (=n). See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_LOG_BACKEND_RTT.html and/or look up
LOG_BACKEND_RTT in the menuconfig/guiconfig interface. The Application Development Primer, Setting
Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful
too.


warning: The choice symbol LOG_BACKEND_RTT_MODE_DROP (defined at subsys/logging/Kconfig:347) was
selected (set =y), but no symbol ended up as the choice selection. See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_LOG_BACKEND_RTT_MODE_DROP.html and/or
look up LOG_BACKEND_RTT_MODE_DROP in the menuconfig/guiconfig interface. The Application Development
Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual
might be helpful too.

-- The C compiler identification is GNU 9.2.0
-- The CXX compiler identification is GNU 9.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /opt/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Configuring done
-- Generating done
-- Build files have been written to: /work/ncs/zephyr/tests/bluetooth/tester/build
-- west build: building application
[1/249] Preparing syscall dependency handling

[244/249] Linking C executable zephyr/zephyr_prebuilt.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:      290352 B         1 MB     27.69%
            SRAM:       53096 B       256 KB     20.25%
        IDT_LIST:         152 B         2 KB      7.42%
[249/249] Linking C executable zephyr/zephyr.elf
```

Reply:
@AyturkDuzen 
That's because you turned CONFIG_TEST back on again in your example here:
```
Loaded configuration '/work/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840_defconfig'
Merged configuration '/work/ncs/zephyr/tests/bluetooth/tester/prj.conf'
Merged configuration '/work/ncs/zephyr/tests/bluetooth/tester/boards/nrf52840dk_nrf52840.conf'
Merged configuration 'prj.conf'
```
prj.conf turns it on, board specific .conf turns it off, and prj.conf (again!) turns it back on.

Don't put `CONFIG_TEST=y` in your overlay config (privacy.conf etc) and you won't have this problem.
I'm assuming privacy.conf only consists of the `CONFIG_BT_PRIVACY=y` or something, if you are duplicating all of the configs in prj.conf then you are doing it wrong.
It should just be additional configurations.

Please remove this commit again, and force push to the branch, don't close the PR.

Reply:
I mean the commit that changes `tests/bluetooth/tester/boards/nrf52840dk_nrf52840.conf`

Reply:
Thanks for catching that. The changes are done on auto-pts repo to not duplicate all the configs from prj.conf (https://github.com/AyturkDuzen/auto-pts/commit/498c59bac49149a46ae98ce4f83d0b3628565dce). PR will be created to intel auto-pts as well. 
Now with omitting the last commit, it works fine without any warning on nrf52840 and nrf5340.